packages/suite/src/views/wallet/nfts/NftsTable/NftsRow.tsx

```diff
@@ -0,0 +1,227 @@
+import { Network } from '@suite-common/wallet-config';
+import { DefinitionType, tokenDefinitionsActions } from '@suite-common/token-definitions';
+import { TokenManagementAction } from '@suite-common/token-definitions';
+import { Token } from '@trezor/blockchain-link-types/src/blockbook-api';
+import {
+    Button,
+    Column,
+    Dropdown,
+    GroupedMenuItems,
+    Icon,
+    Link,
+    Row,
+    Table,
+    TruncateWithTooltip,
+} from '@trezor/components';
+import { Translation } from 'src/components/suite';
+
+import { NftType } from 'src/utils/wallet/nftUtils';
+
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { SUITE } from 'src/actions/suite/constants';
+
+import { useTranslation } from 'src/hooks/suite';
+import styled from 'styled-components';
+import { spacingsPx } from '@trezor/theme';
+import { selectIsCopyAddressModalShown } from 'src/reducers/suite/suiteReducer';
+import { openModal } from 'src/actions/suite/modalActions';
+import { copyToClipboard } from '@trezor/dom-utils';
+import { notificationsActions } from '@suite-common/toast-notifications';
+import { AddressType } from '@suite-common/wallet-types';
+
+type NftsRowProps = {
+    nft: Token;
+    type: NftType;
+    network: Network;
+    shown?: boolean;
+};
+
+const ContractAddress = styled.div`
+    display: inline-block;
+    max-width: 200px;
+    word-break: break-all;
+    white-space: wrap;
+`;
+
+const IconWrapper = styled.div`
+    display: inline-block;
+    margin-left: ${spacingsPx.xxs};
+`;
+
+const NftsRow = ({ nft, type, network, shown }: NftsRowProps) => {
+    const dispatch = useDispatch();
+
+    const { translationString } = useTranslation();
+    const shouldShowCopyAddressModal = useSelector(selectIsCopyAddressModalShown);
+
+    const getNftContractExplorerUrl = (network: Network, nft: Token) => {
```

Comment:
does this differ from `getTokenExplorerUrl`?

Reply:
Yes it's different